prow/config/plugins.yaml

```diff
@@ -505,6 +506,20 @@ plugins:
       - verify-owners
       - welcome
       - wip
+  pingcap/tidb-engine-ext:
+    plugins:
+      - approve
+      - assign
+      - blunderbuss
+      - hold
+      - label
+      - lgtm
+      - owners-label
+      - size
+      - trigger
+      - verify-owners
+      - welcome
+      - wip
```

Comment:
```suggestion
      - assign
      - hold
      - label
      - size
      - trigger
      - welcome
      - wip
```

Reply:
We should change the review flow of this repo without approval without the owners.